### PR TITLE
Allow xtensor-fftw to build with -fno-exceptions

### PR DESCRIPTION
**Pull request by kokarez**
This is the next step after: https://github.com/xtensor-stack/xtensor/pull/1801

### CODE REVIEW

File: include/xtensor-fftw/xtensor-fftw_config.hpp [hunk 1]
```diff
@@ -9,8 +9,27 @@
 #ifndef XTENSOR_FFTW_CONFIG_HPP
 #define XTENSOR_FFTW_CONFIG_HPP
 
+#include <iostream>
```
Comment (SylvainCorlay) [inline include/xtensor-fftw/xtensor-fftw_config.hpp]: Maybe we should only include `<iostream>` if `XTENSOR_FFTW_DISABLE_EXCEPTIONS` is defined.

**Comment by SylvainCorlay** [comment]
I left one more minor comment, but this looks good to me.

Regarding the iostream inclusion, I think that the same thing could be done in xtensor core. (adding a `#include <iostream>` if `XTENSOR_DISABLE_EXCEPTIONS` is set.)

File: include/xtensor-fftw/xtensor-fftw_config.hpp (hunk 1 above)
Comment (kokarez) [reply include/xtensor-fftw/xtensor-fftw_config.hpp]: I will be fine with that, this include will not be need else. But maybe it's overkill ? As you decide I will change

Comment (SylvainCorlay) [reply include/xtensor-fftw/xtensor-fftw_config.hpp]: I think that including all `iostream` just to get xtensor should not be required. I would prefer this to be in the `#if defined` section where you do the `std::cerr`.

Sorry for all the nitpicking!

Comment (kokarez) [reply include/xtensor-fftw/xtensor-fftw_config.hpp]: I do small cl for nitpicking :) I changed the code

**Comment by SylvainCorlay** [comment]
Thanks @kokarez!

**Pull request merged**